scripts/object_hashing/create_object_md5.py

```diff
@@ -0,0 +1,121 @@
+from google.cloud import storage
+import argparse
+import logging
+import subprocess
+
+logging.basicConfig(format="%(levelname)s: %(asctime)s : %(message)s", level=logging.INFO)
+
+def main():
+    description = """Generate md5 for object."""
+    parser = argparse.ArgumentParser(description=description)
+
+    parser.add_argument('-o', '--original_object', required=True, type=str, help='original object for md5 sum calculation')
+    parser.add_argument('-d', '--backup_dir', required=False, type=str, help='directory for creating backup of original object')
+    parser.add_argument('-r', '--requester_pays_project', required=False, default=None, type=str, help='directory for creating backup of original object')
+
+    args = parser.parse_args()
+    
+    original_bucket_name = args.original_object.split("/")[:-1][2] # fc-***
+    original_blob_name = "/".join(args.original_object.split("/")[3:]) # dir/filename.txt
+
+    # if the user provides a backup directory
+    if args.backup_dir:
+        create_backup_object(args.backup_dir, original_bucket_name, original_blob_name, args.requester_pays_project)
+    
+    # always create tmp object
+    create_tmp_object(original_bucket_name, original_blob_name, args.requester_pays_project)
```

Comment:
I think might be better to separate the get md5 part and not have the be called from create_tmp_object. To me that is a little clearer. Also 100% preference and works great as it is.